content/en/docs/tasks/debug/debug-cluster/_index.md

```diff
@@ -14,7 +14,23 @@ problem you are experiencing. See
 the [application troubleshooting guide](/docs/tasks/debug/debug-application/) for tips on application debugging.
 You may also visit the [troubleshooting overview document](/docs/tasks/debug/) for more information.
 
+ Before upgrading to Kubernetes 1.24 , please go through [Urgent Upgrade Notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md#urgent-upgrade-notes).
+
 <!-- body -->
+## Cluster not starting
+
+To start your cluster, run the following command:
+
+```shell
+kubelet --help | grep network-plugin
+```
```

Comment:
This is *not* a command that starts your cluster

Reply:
Thanks @sftim , I'll push the new changes now.

Reply:
@sftim The new changes have been pushed. I've deleted the following command which is wrong and believe it can go without mentioning on "How to start a cluster" since it's error resolving page and not "guide page"

Reply:
This is incorrect
You do not start your cluster with `kubelet --help | grep network-plugin`

/hold
Holding until this is corrected